sdk/storage/Azure.Storage.Blobs/swagger/readme.md

```diff
@@ -1700,4 +1700,13 @@ directive:
     $["x-az-public"] = false;
 ```
 
+### Hide BlobRetentionPolicy.AllowPermanentDelete
```

Comment:
I tried transforming it to [x-az-public]=false but it wouldn't change the type to internal, so I removed it for now.

Reply:
Good call